neptune/exceptions.py

```diff
@@ -13,18 +13,45 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+
 from neptune import envs
 
+STYLES = {'h1': '\033[95m',
+          'h2': '\033[94m',
+          'python': '\033[96m',
+          'bash': '\033[95m',
+          'warning': '\033[93m',
+          'correct': '\033[92m',
+          'fail': '\033[91m',
+          'bold': '\033[1m',
+          'underline': '\033[4m',
+          'end': '\033[0m'}
+
 
 class NeptuneException(Exception):
     pass
 
 
-class Uninitialized(NeptuneException):
+class NeptuneUninitializedException(NeptuneException):
     def __init__(self):
-        super(Uninitialized, self).__init__(
-            "You must initialize neptune-client first. "
-            "For more information, please visit: https://github.com/neptune-ai/neptune-client#initialize-neptune")
+        message = """
+{h1}     
+----NeptuneUninitializedException---------------------------------------------------------------------------------------
+{end}
+You must initialize neptune-client before you create an experiment.
+
+Looks like you forgot to add:
+    {python}neptune.init(api_token='YOUR_LONG_TOKEN', project_qualified_name='WORKSPACE_NAME/PROJECT_NAME'){end}
```

Comment:
Here, assuming we're going with proposing to put the token in the code (which is somewhat unorthodox, even though I understand you strive for simplicity), I'd change:
1. the order of the params - if the user doesn't use named params, they'll get another error
2. the `YOUR_LONG_TOKEN` string. I think I might half-wonder if there's something actually called LONG_TOKEN. Let's use our standard name - `YOUR_API_TOKEN`

Reply:
Done